spinalcordtoolbox/reports/qc.py

```diff
@@ -413,4 +419,13 @@ def normalized(img):
 
     sct.printv('Sucessfully generated the QC results in %s' % qc_param.qc_results)
     sct.printv('Use the following command to see the results in a browser:')
-    sct.printv('open file "{}/index.html"'.format(path_qc), type='info')
+    try:
```

Comment:
The try/except is overkill.

Reply:
ImportError raises if the OS is not detectable